aten/src/ATen/native/native_functions.yaml

```diff
@@ -3916,11 +3916,10 @@
   tags: core
 
 # For normal naming convention this should be `mean.out`. However since we already have `mean.out` we have to rename this.
-# FIXME: fix CI jobs and re-enable this
-#- func: mean.dtype_out(Tensor self, *, ScalarType? dtype=None, Tensor(a!) out) -> Tensor(a!)
-#  device_check: NoCheck   # TensorIterator
-#  dispatch:
-#    CompositeExplicitAutograd: mean_dtype_out
+- func: mean.dtype_out(Tensor self, *, ScalarType? dtype=None, Tensor(a!) out) -> Tensor(a!)
+  device_check: NoCheck   # TensorIterator
+  dispatch:
+    CompositeExplicitAutograd: mean_dtype_out
```

Comment:
you also need to manually author a kernel in  the `at::native::` namespace, with this name (`mean_dtype_out`). Probably in this file: https://github.com/pytorch/pytorch/blob/main/aten/src/ATen/native/ReduceOps.cpp#L1401

Reply:
Yep added